lib/wallaby/browser.ex

```diff
@@ -212,6 +212,10 @@ defmodule Wallaby.Browser do
     path = path_for_screenshot(name: opts[:name])
     File.write! path, image_data
 
+    if opts[:log_screenshot] do
+      IO.puts "Image Screenshot: file:///#{path}"
```

Comment:
Maybe, text could be adjusted to something like.. "Screenshot taken, find it at #{path}" or something like that.

Reply:
I just took the text that Capybara uses, but I don't mind changing it. I'll use the text you recommended.